tests/packages/subproject/pyproject.toml

```diff
@@ -5,3 +5,8 @@
 [build-system]
 build-backend = 'mesonpy'
 requires = ['meson-python']
+
+[tool.meson-python.wheel]
+# Meson versions before 1.3.0 install data files in {datadir}/{project name}/,
```

Comment:
It'd be fine to support this for meson>=1.3.0 only I think.

Reply:
The difference is only in the glob pattern used in the test. It is more work to make the test conditional to the Meson version than to use a glob here.